lib/driver.js

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import { BaseDriver } from 'appium-base-driver';
+import log from './logger';
+import WDAMacServer from './wda-mac';
+import { desiredCapConstraints } from './desired-caps';
+import commands from './commands/index';
+
+const NO_PROXY = [
+  ['GET', new RegExp('^/session/[^/]+/appium')],
+  ['POST', new RegExp('^/session/[^/]+/appium')],
+  ['POST', new RegExp('^/session/[^/]+/element/[^/]+/elements?$')],
+  ['POST', new RegExp('^/session/[^/]+/elements?$')],
+];
+
+class Mac2Driver extends BaseDriver {
```

Comment:
given the large amount of overlap that will exist with the xcui driver, should we have a `BaseWDADriver`?

Reply:
see below